.gitignore

```diff
@@ -10,3 +10,4 @@ src/_drafts
 package-lock.json
 .idea
 .cache
+package.json
```

Comment:
?

Reply:
How to install dependencies if you add `package.json` to `.gitignore` file?

Reply:
Sorry it was pushed by mistake

Reply:
Yes